server/routes/users.js

```diff
@@ -92,4 +92,33 @@ router.post(
     })
 );
 
+// uploads image to AWS s3 and updates user scheme with url of image
+router.post("/profileImageUpload", async function (req, res, next) {
```

Comment:
What is this for, vs the route above it?

Reply:
We don't need that I am not sure how that happened. I think I accidentally accepted both when merging with dev.